plugins/grpc/csharp/v1.65.0/Dockerfile

```diff
@@ -17,10 +17,16 @@ WORKDIR /build/grpc
 RUN bazel build //src/compiler:grpc_plugin_support
 RUN bazel build //src/compiler:grpc_csharp_plugin.stripped
 
+FROM mcr.microsoft.com/dotnet/sdk:6.0.424-bookworm-slim@sha256:39f73995c6d4f64f1a81f4ec42617cb71a797a23da93a5b0109ab06e5094fdae AS dotnetrestore
```

Comment:
We should probably add this image to .github/docker so we can keep it up to date with the desired version of the dotnet SDK.

Reply:
Added a Dockerfile in https://github.com/bufbuild/plugins/pull/1429/commits/88f387bcad94694278536072a84812a31d615ed8 — assuming it'll still pick up updates with the specific sha256 specified?